### ISSUE TITLE
Check the trigger of the eager dispatch ops

### ISSUE THREAD

**Issue by JackCaoG** [ezyang/torchdbg#1]
This is a very cool debugging tool, I was able to see
```
V0715 18:45:24.355000 140534658176832 ../../../usr/local/lib/python3.10/site-packages/torchdbg/__init__.py:171] {"eager_dispatch": {"target": "torch.Tensor.dim", "stack": [{"name": "<module>", "line": 28, "filename": 0, "locals": {}}], "args": [{"dtype": "torch.float32", "shape": [16, 16, 1024, 1024]}], "kwargs": {}, "ret": 4}, "stack": [{"line": 28, "name": "<module>", "filename": 0}, {"line": 1639, "name": "_wrapped_call_impl", "filename": 1}, {"line": 1650, "name": "_call_impl", "filename": 1}, {"line": 160, "name": "forward", "filename": 2}, {"line": 451, "name": "_check_input_dim", "filename": 2}, {"line": 171, "name": "__torch_function__", "filename": 3}]}
V0715 18:45:24.377000 140534658176832 ../../../usr/local/lib/python3.10/site-packages/torchdbg/__init__.py:171] {"eager_dispatch": {"target": "torch.Tensor.add_", "stack": [{"name": "<module>", "line": 28, "filename": 0, "locals": {}}], "args": [{"dtype": "torch.int64", "shape": []}, 1], "kwargs": {}, "ret": {"dtype": "torch.int64", "shape": []}}, "stack": [{"line": 28, "name": "<module>", "filename": 0}, {"line": 1639, "name": "_wrapped_call_impl", "filename": 1}, {"line": 1650, "name": "_call_impl", "filename": 1}, {"line": 173, "name": "forward", "filename": 2}, {"line": 171, "name": "__torch_function__", "filename": 3}]}
V0715 18:45:24.716000 140534658176832 torch/_logging/structured.py:19] {"str": ["/workspaces/dk4/pytorch/torch/nn/functional.py", 4]}
V0715 18:45:24.716000 140534658176832 torch/_logging/structured.py:19] {"str": ["/workspaces/dk4/pytorch/torch/overrides.py", 5]}
V0715 18:45:24.716000 140534658176832 ../../../usr/local/lib/python3.10/site-packages/torchdbg/__init__.py:171] {"eager_dispatch": {"target": "torch.nn.functional.batch_norm", "stack": [{"name": "<module>", "line": 28, "filename": 0, "locals": {}}], "args": [{"dtype": "torch.float32", "shape": [16, 16, 1024, 1024]}, {"dtype": "torch.float32", "shape": [16]}, {"dtype": "torch.float32", "shape": [16]}], "kwargs": {"weight": {"dtype": "torch.float32", "shape": [16]}, "bias": {"dtype": "torch.float32", "shape": [16]}, "training": true, "momentum": 0.1, "eps": 1e-05}, "ret": {"dtype": "torch.float32", "shape": [16, 16, 1024, 1024]}}, "stack": [{"line": 28, "name": "<module>", "filename": 0}, {"line": 1639, "name": "_wrapped_call_impl", "filename": 1}, {"line": 1650, "name": "_call_impl", "filename": 1}, {"line": 193, "name": "forward", "filename": 2}, {"line": 2797, "name": "batch_norm", "filename": 4}, {"line": 1715, "name": "handle_torch_function", "filename": 5}, {"line": 171, "name": "__torch_function__", "filename": 3}]}
```
with a simple `batch_norm`
```
	m = nn.BatchNorm2d(16).to(device)
	m.train()
	input = torch.randn(16, 16, 1024, 1024, device=device)
	
	# warm up compilation cache
	with torchdbg.LoggingMode():
	  input = m(input)
```

I am wondering if there is a way for me to figure out the trigger of those `eager_dispatch` ops. For example on above ops, I saw there is a call to `torch.Tensor.dim` and `torch.Tensor.add_` but I don't know where they are from and why are called. This additional information will be very useful for the debug.

**Comment by ezyang** [comment]
You can get this info by tweaking the "uninteresting" stack frame heuristic https://github.com/ezyang/torchdbg/blob/009ab4364c648a9d41e14f58ca70dd788e7fcdd6/python/torchdbg/__init__.py#L23-L26 

Maybe add a boolean flag that controls whether or not we keep recursing.
